web/angular-wallet/proxy.config.json

```diff
@@ -1,12 +1,13 @@
 {
   "/burst": {
     "target": {
-      "host": "localhost",
+      "host": "75.100.126.226",
       "protocol": "http:",
-      "port": 8125
+      "port": 6876
```

Comment:
testnet

Reply:
Should we make this configurable?

Reply:
technically, it is configurable via this file :P But good idea, I'll check into that.